systemtest/src/test/java/io/strimzi/systemtest/report/DefaultClusterReportST.java

```diff
@@ -0,0 +1,448 @@
+            "kafkaconnects.kafka.strimzi.io.yaml",
+            "kafkamirrormaker2s.kafka.strimzi.io.yaml",
+            "kafkarebalances.kafka.strimzi.io.yaml",
+            "kafkas.kafka.strimzi.io.yaml",
+            "kafkatopics.kafka.strimzi.io.yaml",
+            "kafkausers.kafka.strimzi.io.yaml",
+            "strimzipodsets.core.strimzi.io.yaml"
+        )) {
+            assertValidYamls(outPath + "/reports/customresourcedefinitions", CustomResourceDefinition.class, s, 1);
+        }
+    }
+
+    private void assertValidCustomDeployments(String outPath, String clusterName) throws IOException {
+        for (String s : Arrays.asList(
+            "my-bridge-bridge.yaml",
+            clusterName + "-cruise-control.yaml",
+            clusterName + "-entity-operator.yaml",
+            "strimzi-cluster-operator.yaml"
+        )) {
+            assertValidYamls(outPath + "/reports/deployments", Deployment.class, s, 1);
+        }
+    }
+
+    private void assertValidNetworkPolicies(String outPath, String clusterName) throws IOException {
+        for (String s : Arrays.asList(
+            clusterName + "-entity-operator.yaml",
+            clusterName + "-network-policy-cruise-control.yaml",
+            clusterName + "-network-policy-kafka.yaml",
+            clusterName + "-network-policy-zookeeper.yaml",
+            "my-mm2-mirrormaker2.yaml"
+        )) {
+            assertValidYamls(outPath + "/reports/networkpolicies", NetworkPolicy.class, s, 1);
+        }
+    }
+
+    private void assertValidPodDisruptionBudgets(String outPath, String clusterName) throws IOException {
+        for (String s : Arrays.asList(
+            "my-bridge-bridge.yaml",
+            clusterName + "-kafka.yaml",
+            clusterName + "-zookeeper.yaml",
+            "my-connect-connect.yaml",
+            "my-mm2-mirrormaker2.yaml"
+        )) {
+            assertValidYamls(outPath + "/reports/poddisruptionbudgets", PodDisruptionBudget.class, s, 1);
+        }
+    }
+
+    private void assertValidPods(String outPath, String clusterName) throws IOException {
+        for (String s : Arrays.asList(
+            clusterName + "-kafka-0.yaml",
+            clusterName + "-kafka-1.yaml",
+            clusterName + "-kafka-2.yaml",
+            clusterName + "-zookeeper-0.yaml",
+            clusterName + "-zookeeper-1.yaml",
+            clusterName + "-zookeeper-2.yaml"
+        )) {
+            assertValidYamls(outPath + "/reports/pods", Pod.class, s, 1);
+        }
+        assertValidYamls(outPath + "/reports/pods", Pod.class, "my-bridge-bridge", 1);
+        assertValidYamls(outPath + "/reports/pods", Pod.class, clusterName + "-cruise-control", 1);
+        assertValidYamls(outPath + "/reports/pods", Pod.class, clusterName + "-entity-operator", 1);
+        assertValidYamls(outPath + "/reports/pods", Pod.class, "my-connect-connect", 1);
+        assertValidYamls(outPath + "/reports/pods", Pod.class, "my-mm2-mirrormaker2", 1);
```

Comment:
why these are out of the loop so not in the Arrays.asList declaration as done for other tests?
Apart from the next one which has num=2, they could be moved there.